modules/apps/fragment/fragment-entry-processor/fragment-entry-processor-editable/src/main/java/com/liferay/fragment/entry/processor/editable/EditableDocumentFragmentEntryProcessor.java

```diff
@@ -203,7 +203,8 @@ else if (_fragmentEntryProcessorHelper.isMapped(
 				element.removeAttr("view-tag-name");
 			}
 
-			if (FeatureFlagManagerUtil.isEnabled("LPD-39437") &&
+			if (FeatureFlagManagerUtil.isEnabled(
```

Comment:
@jkappler it is not failing in master, so it is something that we should double check